sonar-dotnet-shared-library/src/main/java/org/sonar/plugins/dotnet/tests/CoverageAggregator.java

```diff
@@ -30,6 +29,9 @@
 import org.sonar.api.utils.log.Logger;
 import org.sonar.api.utils.log.Loggers;
 
+/**
+ * Aggregate the coverage results from different reports of potentially different tools (e.g. aggregate a NCover 3 report with a DotCover one and 3 Visual Studio ones).
```

Comment:
and `OpenCoverReportParser` ... you could remove the names of the tools with something like `see '*ReportParser' fields used by the class`

is it language specific or both C# and VB.NET?